kubetest2-gke/deployer/options/build.go

```diff
@@ -9,16 +9,12 @@ import (
 type BuildOptions struct {
 	Strategy      string `flag:"~strategy" desc:"Determines the build strategy to use either make or bazel."`
 	StageLocation string `flag:"~stage" desc:"Upload binaries to gs://bucket/ci/job-suffix if set"`
-	RepoRoot      string `flag:"~repo-root" desc:"Path to root of the kubernetes repo. Used with --build."`
-
+	RepoRoot      string `flag:"-"`
```

Comment:
Could you explain how `-` works here? Is this change related to the other `RepoRoot` field added in the `deployer` struct?

Reply:
Yup, `-` means this field will not be exposed as a flag: https://github.com/octago/sflags#options-for-flag-tag
now that the top level field is exposed.